src/main/k/working/imp/save-proofs/sum-spec.save-proofs-golden

```diff
@@ -0,0 +1,128 @@
+claim
```

Comment:
Should this be a full Kore definition, so we can have multiple claims?

Reply:
I was just doing this in the next PR, but I guess I could move it here.

Reply:
Done.